open-reac/src/main/java/com/powsybl/openreac/parameters/input/json/OpenReacParametersDeserializer.java

```diff
@@ -20,13 +21,123 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+
+import static java.util.Map.entry;
 
 /**
  * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
  */
 
 public class OpenReacParametersDeserializer extends StdDeserializer<OpenReacParameters> {
 
+    private interface IOBiConsumer<T, U> {
+        void accept(T t, U u) throws IOException;
+    }
+
+    private static BiConsumer<JsonParser, OpenReacParameters> safeRead(IOBiConsumer<JsonParser, OpenReacParameters> consumer) {
+        return (parser, parameters) -> {
+            try {
+                parser.nextToken();
+                consumer.accept(parser, parameters);
+            } catch (IOException e) {
+                throw new PowsyblException(e);
+            }
+        };
+    }
+
+    private static final Map<String, BiConsumer<JsonParser, OpenReacParameters>> FIELD_PROCESSORS = Map.ofEntries(
+            entry("version", (parser, parameters) -> { }),
+            entry("specificVoltageLimits", safeRead((parser, parameters) ->
+                parameters.addSpecificVoltageLimits(parser.readValueAs(new TypeReference<List<VoltageLimitOverride>>() { }))
+            )),
+            entry("variableShuntCompensators", safeRead((parser, parameters) ->
+                parameters.addVariableShuntCompensators(parser.readValueAs(new TypeReference<List<String>>() { }))
+            )),
+            entry("constantQGenerators", safeRead((parser, parameters) ->
+                parameters.addConstantQGenerators(parser.readValueAs(new TypeReference<List<String>>() { }))
+            )),
+            entry("variableTwoWindingsTransformers", safeRead((parser, parameters) ->
+                parameters.addVariableTwoWindingsTransformers(parser.readValueAs(new TypeReference<List<String>>() { }))
+            )),
+            entry("configuredReactiveSlackBuses", safeRead((parser, parameters) ->
+                parameters.addConfiguredReactiveSlackBuses(parser.readValueAs(new TypeReference<List<String>>() { }))
+            )),
+            entry("objective", safeRead((parser, parameters) ->
+                parameters.setObjective(OpenReacOptimisationObjective.valueOf(parser.getText()))
+            )),
+            entry("objectiveDistance", safeRead((parser, parameters) ->
+                parameters.setObjectiveDistance(parser.getValueAsDouble())
+            )),
+            entry("logLevelAmpl", safeRead((parser, parameters) ->
+                parameters.setLogLevelAmpl(OpenReacAmplLogLevel.valueOf(parser.getText()))
+            )),
+            entry("logLevelSolver", safeRead((parser, parameters) ->
+                parameters.setLogLevelSolver(OpenReacSolverLogLevel.valueOf(parser.getText()))
+            )),
+            entry("debugDir", safeRead((parser, parameters) ->
+                parameters.setDebugDir(parser.readValueAs(new TypeReference<String>() { }))
```

Comment:
Previously, it was `parameters.setDebugDir(parser.getText());`.

So, this line instead:
```suggestion
                parameters.setDebugDir(parser.getText())
```